src/test/neorados/test_neorados.cc

```diff
@@ -44,7 +44,7 @@ class TestNeoRADOSCoro : public ::testing::Test {
 TEST_F(TestNeoRADOS, MakeWithLibRADOS) {
   librados::Rados paleo_rados;
   auto result = connect_cluster_pp(paleo_rados);
-  ASSERT_EQ("", result);
+  EXPECT_EQ("", result);
```

Comment:
You can use `ASSERT_*` here. The only place you can't use it is inside a C++20 coroutine.